pkg/machine/wsl/machine.go

```diff
@@ -481,6 +481,7 @@ func runCmdPassThrough(name string, arg ...string) error {
 	cmd.Stdin = os.Stdin
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
+	cmd.Env = []string{"WSL_UTF8=1"}
```

Comment:
This is going to unset all other environment variables.
You must use something like `cmd.Env = append(os.Environ(), "_UNSHARED=true")`